core/domain/html_validation_service.py

```diff
@@ -749,3 +753,51 @@ def _validate_customization_args_in_tag(tag):
                             yield err_msg
     except Exception as e:
         yield str(e)
+
+
+def add_dimensions_to_noninteractive_image_tag(exp_id, html_string):
+    """Adds dimensions to all oppia-noninteractive-image tags.
+
+    Args:
+        html_string: str. HTML string in which the dimensions is to be
```

Comment:
html_string: str. The HTML string to modify; it may or may not contain image tags.

Reply:
Done